styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
+  .text37{
+    font-size: 37px;
+    line-height: 24px;
+  }
+  &, *{
+    @extend .white;
+  }
+  .contactLink{
```

Comment:
A more module name would be better here too. A single class is enough to define this property instead of nested class names.